.pipelines/generate-release-notes-tagged.yml

```diff
@@ -1,42 +0,0 @@
-# Azure DevOps Pipeline for generating release notes
```

Comment:
This is not part of our current Ev2 release process, but I think there was discussion that it should be, yes? We should have Releases in GitHub that include public release notes of what we are releasing?